packages/opentelemetry-sdk-node/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenTelemetry SDK for NodeJS
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+@ToDo
```

Comment:
Will these todos be addressed in this PR? If not, maybe we can use HTML comments to indicate there are works still need to be done.

Reply:
Could be in this PR, but I was going to do in a follow up if everyone is ok with that.

Reply:
I'd +1 👍  if the TODO will be commented, i.e. do not show up in the GitHub rendered results.